Prolific/Prolific/View Controllers/Home View Controller/HomeViewController.m

```diff
@@ -55,6 +56,22 @@ - (void)viewDidLoad {
     [self loadProjects];
 }
 
+- (void)loadCurrentUser {
```

Comment:
hmm do you still need to load current user if the user is now passed in? or am I missing something